site/en/docs/extensions/migrating/remote-hosted-code/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Replace remote hosted code
+subhead: 'Alternatives to remote hosted code in MV3'
+description: 'Different ways to migrate remote hosted code to MV3.'
+date: 2023-08-04
+---
+
+{% Partial 'extensions/mv3-support.md' %}
+
+This section provides an overview of different strategies for migrating MV2 extensions relying on remote hosted code to Manifest V3.
+
+Manifest V3 removes the ability for an extension to use [remotely hosted code][1], which presents security risks by allowing unreviewed code to be executed in extensions. With this change, an extension can only execute JavaScript that is included within its package and subject to review by the Chrome Web Store.
```

Comment:
Nit: This reads to me like the change in MV3 is what's causing the security risk. I know that isn't what you meant, but maybe we could rephrase to make that clearer.

Reply:
rephrased